docker-entrypoint.sh

```diff
@@ -1,27 +1,31 @@
 #!/bin/sh
 set -e
 
+# Default to node user, which is 1000:1000
+export PUID=${PUID:-1000}
+export PGID=${PGID:-1000}
+
 # Function to check if all native modules are working
 check_native_modules() {
     echo "🔍 Checking native modules..."
-    
+
```

Comment:
apologies for all the whitespaces changes 🫣 

I have my editor configured to strip trailing whitespace automatically and didn't inspect the diff locally. I can revert if you'd like